modules

```diff
@@ -3,5 +3,5 @@
 GLUON_SITE_FEEDS='fffd_packages'
 
 PACKAGES_FFFD_PACKAGES_REPO=git://github.com/freifunk-fulda/fffd-packages.git
-PACKAGES_FFFD_PACKAGES_COMMIT=a8e5475d9703ee6e7500025f873b2a52cff68b8a
-PACKAGES_FFFD_PACKAGES_BRANCH=development
+PACKAGES_FFFD_PACKAGES_COMMIT=e6b41d62590192d200f9c31781cbc62a71011b9a
+PACKAGES_FFFD_PACKAGES_BRANCH=issues/78
```

Comment:
Nach dem merge des entsprechenden pull requests in fffd-packages, kann und muss das auch wieder development heißen.